train.py

```diff
@@ -197,14 +197,15 @@ def trainEpoch(epoch):
             total_words += num_words
             report_words += num_words
             if i % opt.log_interval == 0 and i > 0:
-                print("Epoch %2d, %5d/%5d batches; perplexity: %6.2f; %3.0f Source tokens/s; %6.0f s elapsed" %
-                      (epoch, i, len(trainData),
-                      math.exp(report_loss / report_words),
-                      report_src_words/(time.time()-start),
-                      time.time()-start_time))
-
-                report_loss = report_words = report_src_words = 0
-                start = time.time()
+                try:
+                  ans = math.exp(report_loss / report_words)
```

Comment:
Could you use a more meaningful variable name (like `perplexity`)? Also please use the same indentation as the rest of the project (4 spaces).